docs/dump_javadump.md

```diff
@@ -24,13 +24,31 @@
 
 # Java dump
 
-Java dumps, sometimes referred to as *Java cores*, are produced when the VM ends unexpectedly because of an operating system signal, `OutOfMemoryError` exception, or a user-initiated keystroke combination. You can also generate a Java dump by calling the Dump API programmatically from your application or specifying the `-Xdump:java` option on the command line.
+Java dumps, sometimes referred to as *Java cores*, are produced when the VM ends unexpectedly because of an operating system signal, `OutOfMemoryError`, or a user-initiated keystroke combination. You can also generate a Java dump by calling the Dump API programmatically from your application or specifying the `-Xdump:java` option on the command line.
+
+If your Java application crashes or hangs, Java dumps can provide useful information to help you diagnose the root cause.
+
+- If your application crashes, Javadumps are generated automatically for the following types of failure:
+    - the VM receives an unexpected signal or an assertion failure
+    - the VM runs out of memory
+- If your application hangs, you can trigger the generation of a Java dump by sending a SIGSEGV signal (`kill -3`) to the VM.
```

Comment:
I think `SIGSEGV` should be `SIGQUIT`.  We don't want people to send `SIGSEGV` as it makes it look like a crash rather than a user generated javacore

Reply:
Yes, that should definitely say `SIGQUIT`

Reply:
Thanks.  Fixed.